x-pack/platform/plugins/shared/task_manager/server/lib/get_first_run_at.ts

```diff
@@ -24,10 +24,13 @@ export function getFirstRunAt({
 
   if (taskInstance.schedule?.rrule && rruleHasFixedTime(taskInstance.schedule.rrule)) {
     try {
+      const dtstart = taskInstance.schedule.rrule.dtstart
+        ? new Date(taskInstance.schedule.rrule.dtstart)
+        : now;
       const rrule = new RRule({
         ...taskInstance.schedule.rrule,
         bysecond: [0],
-        dtstart: now,
+        dtstart,
       });
       return rrule.after(now)?.toISOString() || nowString;
```

Comment:
This override is not reflected in the `dtstart` returned in the list API, right?

Reply:
Correct, that `dtstart` should remain fixed